src/helpers/region.js

```diff
@@ -0,0 +1,21 @@
+
+export const region = (country) => {
+    switch (country) {
+        case "Europe":
+            return "europe"
+        break;
```

Comment:
In een switch case hoef je na een return nooit meer een break te doen, omdat dat onbereikbare code is vanwege de return
```suggestion
```